src/postgresdb-item.ts

```diff
@@ -17,6 +16,7 @@ export async function getPostgresDbItem(req: Request, res: Response) {
 
   try {
     const client = await pool.connect();
+    await client.query('CREATE TABLE IF NOT EXISTS "items" ("id" SERIAL PRIMARY KEY,"title" varchar(30),"content" varchar(30));');
```

Comment:
split getPostgersDbItem into two function - getPostgresDbItem and listPostgresDbItems

Reply:
what should each function do?

Reply:
got it